### DISCUSSION TITLE
用的是vuepress的啥主题？挺好看的

### DISCUSSION THREAD

**Discussion by nibazshab** [NakanoMikuOrg/arch-guide#54] [Q&A] [answered]
https://arch.icekylin.online

挺好看的，我也想整一个这个主题

**Comment by ice-kylin** [comment, marked as answer]
只是拿官方主题简单改了一下样式表，样式表可以在[这里](https://github.com/NakanoMikuOrg/arch-guide/blob/main/docs/.vuepress/styles/index.scss)找到 😃。
